### DISCUSSION TITLE
运行官网示例上的代码, 能收发成功, 但是查看消息状态为 UNKNOWN, 也有异常消息

### DISCUSSION THREAD

**Discussion by lyingfromyou** [apache/rocketmq#6807] [General]
centos 7
rocket mq version: 5.1.1

运行快速开始中的代码, 收发都能成功, 查看控制台, 消息里面状态为UNKNOWN, 异常信息如下

![errir](https://github.com/apache/rocketmq/assets/24619864/c0016a19-4e6f-412b-b228-bc221f9d7b9b)

**Comment by bugCats** [comment]
要在主题中，添加一个 `%RETRY%[消费组]` 的读写主题。
比喻：你的消费者组名为**qwer**，那么需要添加一个`%RETRY%qwer`的 topic。

**Reply by lyingfromyou** [reply]
加了%RETRY% topic 确实没有异常了
1. 就是说, 我创建topic时, 需要建一个topic, 还要另外建另外一个 %RETRY% topic 吗? 
2. 我能在官网的哪里看到这部分的说明吗?

**Reply by bugCats** [reply]
~内容缺失，文档暂无~
异常提示内容是`没有找到topic=%RETRY%qwer`，点开提示里面的FAQ页面，里面有解决方法。
如果主题重试次数大于1，创建一个topic、消费组，就要同时创建对应的重试主题。

**Comment by cserwen** [comment]
除了 @bugCats  提到的，事实上，这里的消息消费状态不是很准确的，具体可以结合消费堆积 + 消息轨迹的方式来确定消息是否消费成功；这里只是根据消费位点以及消费者的状态进行了一个简单的消费状态展示，比如消费失败了，这里只会显示consumed，无法表示消费成功还是失败。 @lyingfromyou

**Reply by lyingfromyou** [reply]
好的, 感谢
